addons/product/views/res_config_settings_views.xml

```diff
@@ -15,6 +15,14 @@
                         </setting>
                     </block>
                 </xpath>
+                <setting id="module_auth_oauth" position="after">
+                    <setting id="process_barcode_lookup_setting" string="Barcode Database" documentation="https://www.barcodelookup.com/api-documentation">
+                        <field name="module_product_barcodelookup"/>
```

Comment:
If defined in community, it should be defined as ` widget="upgrade_boolean"`, but do we really want to publicize the module here for community users ?